Extension/package.json

```diff
@@ -483,11 +475,8 @@
           "scope": "resource"
         },
         "C_Cpp.default.enableConfigurationSquiggles": {
-          "type": [
-            "boolean",
-            "null"
-          ],
-          "default": null,
+          "type": "boolean",
+          "default": false,
```

Comment:
I thought "null" was equivalent to false.

Reply:
This should be `true`

Reply:
FYI, the prior behavior was for this to be null by default, which was equivalent to false.  I'll change the default to true.